packages/now-cli/src/util/client.ts

```diff
@@ -63,19 +63,19 @@ export default class Client extends EventEmitter {
       : '';
 
     if (opts.accountId || opts.useCurrentTeam !== false) {
-      const query = parsedUrl.query;
+      const query = new URLSearchParams(parsedUrl.query);
```

Comment:
I think there are some small differences between `querystring` and `URLSearchParams` but this will probably be fine for our purposes.

Reply:
@AndyBitz I just realized you can probably use `new URL()` here too instead of `parseUrl(_url, true)`

Reply:
Not in this case unfortunately, because `new URL()` requires a complete URL (including host and protocol), `url.parse` works with just a pathname.